error.js

```diff
@@ -3,10 +3,25 @@ exports.handlePsqlError = (err, req,res, next) => {
     else next(err)
 }
 
-exports.invalidRouteHandler = (req,res,next) => {
-    console.log("sfsdfasdf")
-    // console.log(err, 'asddfsd')
-   
-    res.status(404).send({error:"Route doesn't exist"})
+exports.invalidRouteHandler = (err,req,res,next) => {
+    if(err.status == 404){
+        res.status(404).send({error:"Route doesn't exist"})
+    }else{
+        next(err)
+    }
```

Comment:
The handler for a "route not found" actually can be a regular middleware function that simply sends a 404 straight away